flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/InputGateFairnessTest.java

```diff
@@ -266,13 +281,14 @@ public void testFairConsumptionRemoteChannels() throws Exception {
 	//  Utilities
 	// ------------------------------------------------------------------------
 
-	private SingleInputGate createFairnessVerifyingInputGate(int numberOfChannels) {
+	private SingleInputGate createFairnessVerifyingInputGate(int numberOfChannels, NetworkBufferPool bufferPool) {
```

Comment:
do we need to pass `bufferPool` here always as argument if it is a class field?

Reply:
Not need this field actually.  In my first version for this test it was not a class field. After refactoring to class field I forgot to change this part.